### ISSUE TITLE
Rounded corners in the bottom "leak" some elements like the dashed line, scrollbar and the overscroll animation.

### ISSUE THREAD

**Issue by pedrofleck** [vinceliuice/Orchis-theme#52]
The rounded corners in this theme look awesome, however, since it is some "hack" of how GTK3 windows usually look, some elements still act like the corners in the bottom of the window were squared.
For example, in this screenshot I've taken in Settings, the overscroll animation "leaks" outside of the window on the bottom right, this also happens with the dashed line (the one that shows there's more content on a list) on the bottom left. When I click in the scrollbar, it also shows outside of the window.
![orchis](https://user-images.githubusercontent.com/761321/103785660-3d22e180-501a-11eb-9d0e-7a451ed17c5c.png)
PS: I added the green background to make it easier to see without motion.

**Comment by vinceliuice** [comment]
Indeed, I still have no way to solve this problem. I hope that the official gnome can add support for rounded corners in the future.

**Comment by a1eaiactaest** [comment]
how can i get rounded corners to ubuntu yaru's defaults while still using Orchis theme
